java-tools/OsmAndMapCreatorUtilities/src/main/java/net/osmand/obf/OsmGpxWriteContext.java

```diff
@@ -107,6 +107,7 @@ public void writeTrack(OsmGpxFile gpxInfo, Map<String, String> extraTrackTags, G
 				serializer.attribute(null, "lon", latLonFormat.format(gpxFile.findPointToShow().lon));
 				tagValue(serializer, "route", "segment");
 				tagValue(serializer, "route_type", "track");
+				tagValue(serializer, "route_radius", gpxFile.getOuterRadius());
```

Comment:
we should have limited number of values: 5, 10, 50, 100, 500, 1000, 5000, 10000

Reply:
A - 5, B - 10, C - 50, D - 100,  E - 500, F - 1000, G - 5000, H -  10000

( (int) (10 ^ ((X - 'A' + 1) / 2) * 5 + 1) / 5)

Reply:
int d = x - 'a' + 1;
int r = (int) Math.pow(10, ((x - 'a' + 1) / 2)) * (d % 2 == 0 ? 1 : 5);
System.out.println(x + " " + r);